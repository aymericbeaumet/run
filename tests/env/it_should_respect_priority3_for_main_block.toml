env = [ "__RUN_CLI_FOOBAR__=from_main_block" ]

[[run]]
name = "p3"
cmd = ["bash", "-c", "echo $__RUN_CLI_FOOBAR__"]