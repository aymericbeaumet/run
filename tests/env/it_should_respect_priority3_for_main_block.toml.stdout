[p3] from_main_block